bash_completions.go

```diff
@@ -469,6 +551,9 @@ func writeRequiredNouns(buf *bytes.Buffer, cmd *Command) {
 	for _, value := range cmd.ValidArgs {
 		buf.WriteString(fmt.Sprintf("    must_have_one_noun+=(%q)\n", value))
 	}
+	if cmd.ValidArgsFunction != nil {
```

Comment:
> disallow using both together and cause some configuration error if they are used together.

I couldn't find a good way to send an error when both `ValidArgs` and `ValidArgsFunction` are present.  The closest I could find is here, in `writeRequiredNouns()` , which runs when generating the completion script.  Although there is no error being returned, so all we could do is `log.Fatal()`. 

Any suggestions @jharshman ?

Reply:
@marckhouzam I'm kinda leaning towards just documenting the behavior rather than bailing out. `ValidArgs` already takes precedence so if it's sufficiently documented then I think it's good enough.

Reply:
ok with me.